### PR TITLE
Add text

### PR DESCRIPTION
**Pull request by natsmnv**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,6 +6,6 @@
     <title>Document</title>
 </head>
 <body>
-    <p></p>
+    <p>Hello</p>
```
Comment (YershovArt) [inline index.html:9]: add "World"

**Review by YershovArt — CHANGES_REQUESTED** [review]

**Comment by ivan-solonchenko** [comment]
add div and then p

**Comment by natsmnv** [comment]
Done

**Review by YershovArt — APPROVED** [review]
done

**Pull request merged**